README.md

```diff
@@ -67,10 +67,10 @@ response in a text view.
 You can build the application by running:
 
 ```
-$ bazel build //ios-app:GetUrl
+$ bazel build ios-app --ios_sdk_version=8.4
```

Comment:
honestly, I'm not really sure why this worked before, perhaps default behavior changed? I was getting complaints about UiKit not being supported with my current default sdk version, so this takes out the guesswork.

If this is a not-ideal step, I can delve deeper and figure out how to have the app supported with the default sdk version

Alternatively, do you want me to explain this flag in the readme?